sigle/src/modules/publicHome/components/PublicHome.tsx

```diff
@@ -232,32 +266,109 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
                   : `@${twitterHandle}`}
               </Typography>
             )}
+            <Box
+              css={{ width: '1px', height: '$4', backgroundColor: '$gray9' }}
+            />
+            <Flex
+              align="center"
+              gap="2"
+              css={{
+                '&:hover': {
+                  '& button': {
+                    display: 'block',
+                  },
+                },
+              }}
+            >
+              <Typography
+                size="subheading"
+                css={{
+                  color: '$gray9',
+
+                  '&:hover': {
+                    color: '$gray10',
+                  },
+                  '&:active': {
+                    color: '$gray12',
+                  },
+                }}
+              >
+                {abbreviateAddress(userInfo.address)}
```

Comment:
I think this link should be clickable and open a new tab showing the address in the stacks explorer, wdyt?